library/components/TPdf.vue

```diff
@@ -1,33 +1,30 @@
 <template>
-    <div class="relative" @mouseover="handleTooltip" @mouseleave="showPdfTooltip = false">
+    <div class="relative">
         <!-- PDF tooltip -->
-        <t-tooltip v-if="showPdfTooltip" position="left" width="400px">
+        <t-tooltip position="left" width="400px" is-clickable>
```

Comment:
after clicking the button to download the pdf, the tooltip never comes back.
https://user-images.githubusercontent.com/1307818/186701000-7e7f6bf8-1cab-4a2e-946a-c09c3e061033.mov

Reply:
In the case of t-pdf I am using the `is-clickable` prop so it only shows up on click, but this will confusing isn't it, yes should be handled better I think.

Reply:
I am used to tooltips behaving like this: https://dequeuniversity.com/library/aria/tooltip

I think hovers and focus are enough for our uses cases, no?

Reply:
@sareg0  I see, I like these examples, if hovered on a button, then hide on `mouseleave` but if focused, keep it until user clicks away.